cli/commands/common.go

```diff
@@ -86,3 +86,22 @@ func addReplicasetFlag(cmd *cobra.Command) {
 func setStateboardFlagIsSet(cmd *cobra.Command) {
 	ctx.Running.StateboardFlagIsSet = cmd.Flags().Changed("stateboard")
 }
+
+func setFailoverFlagsIsSet(cmd *cobra.Command) {
+	ctx.Failover.StateProviderIsSet = cmd.Flags().Changed("state-provider")
+	ctx.Failover.FailoverTimeoutIsSet = cmd.Flags().Changed("failover-timeout")
+	ctx.Failover.FencingEnabledIsSet = cmd.Flags().Changed("fencing-enabled")
+	ctx.Failover.FencingTimeoutIsSet = cmd.Flags().Changed("fencing-timeout")
+	ctx.Failover.FencingPauseIsSet = cmd.Flags().Changed("fencing-pause")
+}
+
+func addCommonFailoverParamsFlags(cmd *cobra.Command) {
+	cmd.Flags().IntVar(&ctx.Failover.FailoverTimeout, "failover-timeout", 0, failoverTimeoutUsage)
```

Comment:
Seems that it can also be JSON'like params. `--params '{"failover_timeout"}'` and so on. It will work like for state provider params. Newly added parameters will be supported and parsing becomes much easier. I don't think somebody would like to use `set` with many flags instead of `setup` for cases when we need many parameters.